### PR TITLE
Add dec2

### PR DESCRIPTION
**Pull request by koverholt**

### CODE REVIEW

**Comment by koverholt** [comment]
Depends on recipe in https://github.com/conda-forge/staged-recipes/pull/626.

**Comment by danielfrg** [comment]
I just uploaded a new `dec2` v0.2.1 to pypi, should work on all python versions now that removed `salt-pepper` as dependency.

**Comment by koverholt** [comment]
Great, thanks! I just updated this recipe to use dec 0.2.1 and updated the dependencies.

**Comment by koverholt** [comment]
This is now ready for review.

**Comment by jakirkham** [comment]
LGTM.

We might want to get your dependencies over here at some point, but that's not a blocker.

Will wait and see if anyone else sees anything needed here.

**Comment by jakirkham** [comment]
Thanks @koverholt.

File: recipes/dec2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - dec2 --help
+
+about:
+  home: https://github.com/dask/dec2
+  license: Apache 2.0
+  summary: 'Start a cluster in EC2 for dask.distributed'
+
+extra:
+  recipe-maintainers:
+    - danielfrg
```
Comment (jakirkham) [inline recipes/dec2/meta.yaml]: Guessing this ok, @danielfrg. Care to confirm though?

Comment (danielfrg) [inline recipes/dec2/meta.yaml]: Yes thats ok.

**Pull request merged**